### PR TITLE
Add support for scylladb/cassnadra

### PR DESCRIPTION
**Pull request by fruch**

### CODE REVIEW

File: fastapi_pagination/ext/cassandra.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+from cassandra.cqlengine.models import Model
+
+from ..api import create_page
+from ..bases import AbstractPage
+from ..utils import TParams, verify_params
+
+T = TypeVar("T", bound=Mapping[str, Any])
+
+
+def paginate_with_cursor(
```
Comment (uriyyo) [inline fastapi_pagination/ext/cassandra.py:14]: Let's call it `paginate` as we do it for other integrations.

File: pyproject.toml [hunk 2]
```diff
@@ -72,7 +73,8 @@ piccolo = ["piccolo"]
 motor = ["motor"]
 mongoengine = ["mongoengine"]
 beanie =  ["beanie"]
-all = ["sqlmodel", "SQLAlchemy", "databases", "orm", "ormar", "tortoise-orm", "asyncpg", "django", "piccolo", "motor", "mongoengine", "pony", "beanie", "sqlakeyset"]
+scylla = ["scylla-driver"]
+all = ["sqlmodel", "SQLAlchemy", "databases", "orm", "ormar", "tortoise-orm", "asyncpg", "django", "piccolo", "motor", "mongoengine", "pony", "beanie", "sqlakeyset", "scylla"]
```
Comment (uriyyo) [inline pyproject.toml:77]: I think it should be `scylla-driver`, not `scylla`.

File: fastapi_pagination/ext/cassandra.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    model: Type[Model],
+    query_filter: Optional[Dict[Any, Any]] = None,
+    params: Optional[TParams] = None,
+    **kwargs: Any,
```
Comment (uriyyo) [inline fastapi_pagination/ext/cassandra.py:18]: `kwargs` is not used, we can remove them.

File: fastapi_pagination/ext/cassandra.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        fetch_size=raw_params.size,
+    )
+    conn = connection.get_connection()
+
+    cursor = conn.session.execute(
+        stmt, parameters={str(i): v for i, v in enumerate(query_filter.values())}, paging_state=raw_params.cursor
+    )
+    items = cursor.current_rows
+
+    return create_page(items, total=0, params=params, next_=cursor.paging_state)
```
Comment (uriyyo) [inline fastapi_pagination/ext/cassandra.py:34]: `total` can be removed, it's not required for `CursorPage`

**Review by uriyyo — CHANGES_REQUESTED** [review]
LGTM, I have only minor suggestions

File: fastapi_pagination/ext/cassandra.py (hunk 1 above)
Comment (fruch) [reply fastapi_pagination/ext/cassandra.py]: changed

File: pyproject.toml (hunk 2 above)
Comment (fruch) [reply pyproject.toml]: changed

File: fastapi_pagination/ext/cassandra.py (hunk 4 above)
Comment (fruch) [reply fastapi_pagination/ext/cassandra.py]: removed

File: fastapi_pagination/ext/cassandra.py (hunk 3 above)
Comment (fruch) [reply fastapi_pagination/ext/cassandra.py]: removed

**Review by uriyyo — APPROVED** [review]

**Comment by fruch** [comment]
@uriyyo I had a small mistake on the tests, dropping un-existed keyspace

**Comment by fruch** [comment]
@uriyyo seems that I've forgotten to update the readme on the Scylla/Cassandra support...

**Comment by uriyyo** [comment]
@fruch Yup, we forgot to do it.

Would you like to submit PR to fix it?

**Pull request merged**